tests/docker/Dockerfile

```diff
@@ -60,7 +60,7 @@ ARG ducker_creator=default
 LABEL ducker.creator=$ducker_creator
 
 # Update Linux and install necessary utilities.
-# we have to install git since it is included in openjdk:8 but not openjdk:11
+# we have to install git since it is not included in openjdk:11
```

Comment:
I think we can remove this comment, as later versions of the container no longer have Git installed.